DocumentedExamples/Atlantic_IndoPacific_Basin_Overturning_Circulation.ipynb

```diff
@@ -8,7 +8,9 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    ax.set_ylim([-80,90])
</pre><p>A semi-colon on the end of this line would prevent the text output above the plot. Same for all of the cells with plots. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/224/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>